Source/WebCore/platform/graphics/win/FontCustomPlatformDataWin.cpp

```diff
@@ -100,6 +100,11 @@ RefPtr<FontCustomPlatformData> createFontCustomPlatformData(SharedBuffer& buffer
     return adoptRef(new FontCustomPlatformData(fontName, WTFMove(creationData)));
```

Comment:
It seems like we could be doing a better job of avoiding the copying of font buffer data in some of these create methods. (Not necessary for this patch -- just an idea for improvement).